src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -515,9 +516,15 @@ CLASS ZCL_ABAPGIT_OBJECT_FUGR IMPLEMENTATION.
         function_pool_not_found = 1
         OTHERS                  = 2.
     IF sy-subrc <> 0.
-      zcx_abapgit_exception=>raise( 'Error from RS_FUNCTION_POOL_CONTENTS' ).
+      zcx_abapgit_exception=>raise( |Error from RS_FUNCTION_POOL_CONTENTS for { lv_area }| ).
     ENDIF.
 
+* The result can also contain function which are lowercase.
+    LOOP AT rt_functab ASSIGNING <ls_functab>.
+      TRANSLATE <ls_functab> TO UPPER CASE.
+      MODIFY rt_functab FROM <ls_functab>.
```

Comment:
can the MODIFY be omitted? TRANLSATE changes the field-symbol which changes the table?

Reply:
It can be ommitted.